### DISCUSSION TITLE
Reducing database usage

### DISCUSSION THREAD

**Discussion by BunyaminAkdeniz** [19/classroom#68] [Anonim]
Merhaba Hocam. Projem için nosql olan firebase'i kullanıyorum. 
![Screenshot 2023-06-11 150416](https://github.com/19/classroom/assets/91963771/546cd865-4247-4726-8a35-523e58f42b33)
Ekranda gördüğünüz bir kullanıcının klasörleri. Başka bir kullanıcı bu klasörlerin bazılarını takip ediyor. Bunu takip ettiği klasörlerin ID'lerini kaydederek yapıyor. Sorum şu ki kullanıcının takip ettiği klasörleri getirirken, takip ettiği klasör id'leri ile tek tek get işlemi mi yapmalıyım yoksa klasör sahibinin tüm klasörlerini getirip id'lere göre filtreleyip mi göstermeliyim ? 

Küçük demo uygulamalarım için local database açıyorum ve yine aynı ikileme düşüyorum. Bu işlemler arasında sitemi yorma açısından, fiyatlandırma açısından, vs. avantajlı olan hangisidir kestiremiyorum. 
@roktas

**Comment by roktas** [comment]
Bu konuda yeterli bilgim yok maalesef. Şimdilik kapatıyorum.
